src/template-tags/tickets.php

```diff
@@ -10,6 +10,46 @@
 	exit( '-1' );
 }
 
+if ( ! function_exists( 'tribe_tickets_parent_post' ) ) {
+	/**
+	 * Returns the current post object that can have tickets attached to it
+	 *
+	 * Optionally the post object or ID of a post can be passed in and,
```

Comment:
Yeah. I was being lazy and copy/pasting the comment from the-events-calendar and removing the word *event*. I'll use your suggestion!